packages/apps/src/initSettings.ts

```diff
@@ -22,6 +22,7 @@ function getApiUrl (): string {
     // https://polkadot.js.org/apps/?rpc=ws://127.0.0.1:9944#/explorer;
     const url = decodeURIComponent(urlOptions.rpc.split('#')[0]);
 
+    assert(url.includes('.cere.network') || url.includes('localhost') || url.includes('127.0.0.1'), 'Invalid ws endpoint');
```

Comment:
@karthikeyan-cere 
We can not use `includes`. An URL should ends with `.cere.network`. Otherwise, anybody can hack this check with `cere.network.xyz.com` or other similar.

Reply:
It should be equal to `localhost`.

Reply:
I suggest using regex and double-check the ticket description.

Reply:
ok